apps/www/components/copy-button.tsx

```diff
@@ -77,6 +77,36 @@ export function CopyButton({
   )
 }
 
+export function CopyButtonForDialog({
+  value,
+  className,
+  src,
+  event,
+  ...props
+}: CopyButtonProps) {
+  const [hasCopied, setHasCopied] = React.useState(false)
+
+  React.useEffect(() => {
+    setTimeout(() => {
+      setHasCopied(false)
+    }, 2000)
+  }, [hasCopied])
+
+  return (
+    <div>
```

Comment:
is not better here use a `Fragment` (`<>`)  instead `div`?
Only suggestion, maybe I'm wrong.

Reply:
Hmm. You have a valid point but since cn are available , i guess it would give them a flexible over block element . what do you think ?